openpype/hosts/houdini/plugins/create/create_pointcache.py

```diff
@@ -57,3 +91,17 @@ def get_network_categories(self):
             hou.ropNodeTypeCategory(),
             hou.sopNodeTypeCategory()
         ]
+
+    def get_obj_output(self, obj_node):
+        """Find output node with the smallest 'outputidx'."""
+
+        outputs = dict()
+
+        for sop_node in obj_node.children():
+            if sop_node.type().name() == 'output' :
+                outputs.update({sop_node : sop_node.parm('outputidx').eval()})
+
+        if outputs:
+            return min(outputs, key = outputs.get)
+        else:
+            return
```

Comment:
This should be using [`ObjNode.subnetOutputs()`](https://www.sidefx.com/docs/houdini/hom/hou/ObjNode.html)

> Return the child output nodes of a subnetwork.
>
> Certain networks, such as SOPs, have special Output nodes to override the canonical Display or Render nodes and allow multiple outputs. If any output nodes are present, this returns a list of those nodes. If no output nodes are present, either the display or render node is returned depending on if the target of cooking is the display or an output driver. If the node has no children, an empty list is returned.

Reply:
I combined both solutions into this :D 
```python 
def get_obj_output(obj_node):
    outputs = obj_node.subnetOutputs()
    
    # if obj_node is empty
    if not outputs: 
        return 
    
    # if obj_node has one output child whether its
    # sop output node or a node with the render flag
    elif len(outputs)==1: 
        return outputs[0] 
    # if there are more than one, then it have multiple ouput nodes
    # return the one with the minimum 'outputidx'
    else:
        return (min(outputs, key=lambda node : node.parm('outputidx').eval()))
```